### PR TITLE
HomePage SEO Description

### PR DESCRIPTION
**Pull request by awayken**
Addresses #199.

### CODE REVIEW

File: web_modules/layouts/HomePage/index.js [hunk 1]
```diff
@@ -39,11 +39,12 @@ export default class HomePage extends Component {
       { property: "og:title", content: head.title },
       { property: "og:type", content: "article" },
       { property: "og:url", content: this.props.__url },
-      // { property: "og:description", content: pageDescription(body) },
+      { property: "og:description", content: head.description },
+
       { name: "twitter:card", content: "summary" },
       { name: "twitter:title", content: head.title },
       { name: "twitter:creator", content: `@${ pkg.twitter }` },
-      // { name: "twitter:description", content: pageDescription(body) },
```
Comment (thangngoc89) [inline web_modules/layouts/HomePage/index.js]: Why remove twitter:description tag ?

**Comment by thangngoc89** [comment]
There is a commented `pageDescription(body)` function in the top of the file. Could you also remove it ? It's from Statinamic's boilerplate.

File: web_modules/layouts/HomePage/index.js (hunk 1 above)
Comment (awayken) [inline web_modules/layouts/HomePage/index.js]: @thangngoc89 In #199 @marcustisater advised me to use https://github.com/postcss/postcss.org/blob/master/web_modules/layouts/Page/index.js for reference, which omits a twitter:description for a normal meta description.

However, now that I review https://dev.twitter.com/cards/types/summary, I see that twitter:description is considered required for Summary Cards. I'll add it.

**Comment by awayken** [comment]
@thangngoc89 Can you quote the line where the `pageDescription(body)` is? I can't see to find it.

**Comment by thangngoc89** [comment]
Ouch. Totally my bad. Looks like those LoC was removed recently.

**Comment by awayken** [comment]
@thangngoc89 I've updated the Twitter Summary Card tags to be compliant with Twitter's documentation.

**Comment by thangngoc89** [comment]
@awayken

> `twitter:site` - The Twitter username used by the given site including the '@'
> `twitter:creator` - The Twitter username of the individual author of the content

Many sites contain both tag with the same content. Let's wait for members' comment

**Comment by awayken** [comment]
@thangngoc89 We could certainly do that. I know from https://dev.twitter.com/cards/types/summary that twitter:site is required, but there's no reason we couldn't have both.

**Comment by marcustisater** [comment]
lgtm

**Pull request merged**